src/Stratis.Bitcoin.Features.Consensus.Tests/TimeSyncBehaviorTest.cs

```diff
@@ -186,7 +186,7 @@ public void AddTimeData_WithLargeSampleSet_ForgetsOldSamples()
             }
 
             var allSamples = new List<int>(inSamples);
-            for (int i = 0; i < TimeSyncBehaviorState.OutboundToInboundWeightRatio; i++)
+            for (int i = 0; i < TimeSyncBehaviorState.OffsetWeightSecurityConstant; i++)
```

Comment:
It seems to me that the logic here changed and therefore this test should not pass

Reply:
Okay, it does pass just by coincidence and bug in test
the assertion is wrong in test and the one on line 177 should fail 
as the logic changed these tests needs to be modified